<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDarwinCoreDialogBase</class>
 <widget class="QDialog" name="QgsDarwinCoreDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load darwin core data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="mFromFileButton">
       <property name="text">
        <string>From file:</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mFileLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mFilePushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="mFromUrlButton">
       <property name="text">
        <string>From url:</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mUrlLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsDarwinCoreDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsDarwinCoreDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
